### DISCUSSION TITLE
Bug on develop

### DISCUSSION THREAD

**Discussion by mejtika** [DevAdventCalendar/DevAdventCalendar#430] [Q&A] [answered]
@patrikulus Probuje sie uporać z bugiem na devie odnośnie braku adventowego czasu. Widzę, że zrobiłeś zmiany w appsettingsach - u mnie lokalnie wszystko dziala zarowno na develop jak i develop_uat. Widzę, że pipeline przechodzi, relase może się opóźniać?

**Comment by WTobor** [comment, marked as answer]
already fixed
